docs/source/install/index.rst

```diff
@@ -3,15 +3,15 @@ Installation
 
 For a quick evaluation of |st2| and package-based installation, start from
 :doc:`Overview <./overview>`, and proceed with the guide for your Linux distribution.
-
+Please note that only 64 bit architecture is currently supported.
 
 .. rubric:: Installations
 
 .. toctree::
     :maxdepth: 1
 
     Reference Deployment <overview>
-    Ubuntu / Debian <deb>
+    Ubuntu / Debian 14.04 (Trusty) <deb>
```

Comment:
Yep. I know about 16.04. When we have CI, I'll edit the docs and mention 16.04. The supported versions doesn't talk about 64 bit and unfortunately a lot of people try 32 bit because Ubuntu downloads are confusing (they call their 64 bit ISOs amd-64). We decided to explicitly call out 64 bit in docs but I am the one making the actual changes